packages/apps/sale/src/screens/saleOrder/SaleQuotationsScreen.tsx

```diff
@@ -21,12 +21,14 @@ import {useTypes} from '@axelor/aos-mobile-core';
 import {Screen} from '@axelor/aos-mobile-ui';
 import {SaleOrderListView} from '../../components';
 
-const SaleQuotationsScreen = ({}) => {
+const SaleQuotationsScreen = ({route}) => {
   const {SaleOrder} = useTypes();
+  const {customer} = route.params;
```

Comment:
Screen from menu entry is broken due to this. 
```
  const {customer} = route.params ?? {};
```
Also please move this above the useTypes call